base/intset.jl

```diff
@@ -220,6 +220,12 @@ function ==(s1::IntSet, s2::IntSet)
     return true
 end
 
+"""
+    issubset(A, S) -> Bool
+    ⊆(A,S) -> Bool
+
+Return `true` if `A` is a subset of or equal to `S`.
```

Comment:
this looks redundant with the more general docstring that you've moved to set.jl ?